test-app/Makefile

```diff
@@ -110,6 +106,22 @@ ifeq ($(TARGET),hifive1)
   APP_OBJS+=hifive1_write_page.o
 endif
 
+# $(CYPRESS_PDL)/devices/templates/COMPONENT_MTB/COMPONENT_CM0P/system_psoc6_cm0plus.o
+
+ifeq ($(TARGET),psoc6)
+  LSCRIPT_TEMPLATE:=ARM-psoc6.ld
```

Comment:
Check indentation here. Some Makefiles require tabs.

Reply:
Fixed, thanks David! In this case it's within ifeq-s and not in a recipe, tabs cannot be used and indentation is just for readability and not enforced by makefile's syntax.